Update README (#5)
meilisearch/integrations-guides: README.md

> # MeiliSearch integrations and sdks guide

Reviewer: ```suggestion
# MeiliSearch integrations and SDKs guide
```

Work in progress:

- [Java](https://github.com/meilisearch/meilisearch-java)

> 

Reviewer: ```suggestion
```

> This repository aims to centralize all informations about the differents sdks & integrations that are built around MeiliSearch.

Reviewer: ```suggestion
This repository aims to centralize all information about the different SDKs & integrations that are built around MeiliSearch.
```

## Documentation

> MeiliSearch documentations is at [docs.meilisearch.com](https://docs.meilisearch.com).

Reviewer: ```suggestion
Here is the [MeiliSearch documentation](https://docs.meilisearch.com).
```

> There is no documentation for the different sdks except the dedicated READMEs. There is a [discussion about it](https://github.com/meilisearch/sdks/issues/3).

Reviewer: ```suggestion
There is no documentation for the different SDKs except the dedicated READMEs. There is a [discussion about it](https://github.com/meilisearch/sdks/issues/3).
```

## Building an integration

> If you want to build an integration for MeiliSearch, you are more than welcome to and we would be happy to help you :)

Reviewer: ```suggestion
If you want to build an integration for MeiliSearch, you are more than welcome to and we would be happy to help you :)

```

> Here are some guidelines that you should follow though, if we want to garantee consistency across the differents integrations and technical quality for the different MeiliSearch users.

Reviewer: ```suggestion
Here are some guidelines that you should follow though, if we want to guarantee consistency across the differents integrations and technical quality for the different MeiliSearch users.
```

> - About testing: Currently, the most tested sdk is the Ruby one but if you want to start writing test you can read what the PHP sdk is testing to get an overview of minimum viable tests that your sdk should support.

Reviewer: ```suggestion
#### About testing

If you want to start writing tests you can read what the PHP SDK is testing to get an overview of minimum viable tests that your SDK should support. The JS and the Ruby SDKs are also good examples of deeper tests.
```

> - README: All the README should follow the same patterns. This reository should hold the README template and at some point we might want to automaticaly update all the readmes from here. 

Reviewer: ```suggestion
#### README

All the README should follow the same patterns. This repository should hold the README template and at some point, we might want to automatically update all the readmes from here. 
```
